reporter/errors.go

```diff
@@ -30,26 +30,29 @@ var ErrInvalidSource = errors.New("parse failed: invalid proto source")
 // about the location in the file that caused the error.
 type ErrorWithPos interface {
 	error
-	// GetPosition returns the source position that caused the underlying error.
+	// GetPosition returns the start source position that caused the underlying error.
 	GetPosition() ast.SourcePos
+	// GetEndPosition returns the end source position that caused the underlying error.
+	GetEndPosition() ast.SourcePos
```

Comment:
Since we're making a backwards-incompatible change here anyway, let's improve the method names. I think just _embedding_ `ast.SourceSpan` might be the way to go. Then the implementation struct can also embed it.

Reply:
done